lib/ansible/galaxy/collection/galaxy_api_proxy.py

```diff
@@ -102,6 +110,7 @@ def get_collection_versions(self, requirement):
     def get_collection_version_metadata(self, collection_candidate):
         # type: (Candidate) -> CollectionVersionMetadata
         """Retrieve collection metadata of a given candidate."""
+        self._offline_is_unsupported()
```

Comment:
Seeing more `assert`-like use of this, I'm inclined to recommend the naming option that does have `assert` in it.